packages/geo/src/lib/geometry/geometry-form-field/geometry-form-field-input.component.ts

```diff
@@ -93,17 +94,46 @@ export class GeometryFormFieldInputComponent implements OnInit, OnDestroy, Contr
    */
   @Input() measure: boolean = false;
 
+  /**
+   * Whether draw control should be active or not
+   */
+  @Input()
+  get drawControlIsActive(): boolean { return this._drawControlIsActive; }
+  set drawControlIsActive(value: boolean) {
+    this._drawControlIsActive = value;
+    if (this.ready === false) {
+      return;
+    }
+    this.deactivateControl();
+    if (!this._drawControlIsActive) {
+      return;
+    } else {
+      this.createDrawControl();
```

Comment:
I don't think a new control should be created everytime. Maybe check if one already exists?

Reply:
done